lib/report_plugins/glucosedistribution.js

```diff
@@ -30,6 +30,22 @@ glucosedistribution.html = function html(client) {
   return ret;
 };
 
+glucosedistribution.css =
+  '#glucosedistribution-overviewchart { \
+    width: 2.8in; \
+    height: 2.8in; \
+    float:left; \
+  } \
+  #glucosedistribution-placeholder .tdborder { \
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Bad escaping of EOL. Use option multistr if needed.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/946152869/issues/source?bid=2452379&fileBranchId=2849128#l39)